### DISCUSSION TITLE
What is the `single_file_name` in Github App installation?

### DISCUSSION THREAD

**Discussion by tiev** [community/community#22925] [New to GitHub]
<p>The API response for getting single Github App installation has a field named <code>single_file_name</code>.</p>
<p>What is this file and what is it for?</p>
<p>Ref: <a href="https://developer.github.com/v3/apps/#get-a-single-installation" rel="nofollow noopener">https://developer.github.com/v3/apps/#get-a-single-installation</a></p>

**Comment by ulisigi** [comment, marked as answer]
<p>Hi <a class="mention" href="/tiev">@tiev</a>,</p>
<p>single_file_name is the placeholder for the file path to which GitHub app is granted permission to manage. Hope this helps.</p>
<p>Happy coding!</p>
